.gitignore

```diff
@@ -4,7 +4,7 @@
 .*
 
 # Downloads
-kafka.tgz
+/.downloads
```

Comment:
Does the forward slash prefix mean only for the top directory of the git checkout?

Reply:
yes.